modules/nw-ovn-ipsec-north-south-enable.adoc

```diff
@@ -0,0 +1,138 @@
+
+[NOTE]
+====
+After you apply the machine config, the Machine Config Operator reboots affected nodes in your cluster to rollout the new machine config.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* You have reduced the size of your cluster MTU by `46` bytes to allow for the overhead of the IPsec ESP header.
+* You have installed the `butane` utility.
+* You have an existing `cacert.p12` certificate bundle to copy to cluster hosts.
+
+.Procedure
+
+As a cluster administrator, you can enable IPsec support external for IPsec endpoints.
+
+. Create an IPsec configuration file named `ipsec-endpoint-config.conf`. The configuration is consumed in the next step. For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/securing_networks/configuring-a-vpn-with-ipsec_securing-networks#configuring-a-vpn-with-ipsec_securing-networks[Configuring a VPN with IPsec].
+
+. Provide your certificate bundle in a file named `cacert.p12`. This file is imported as part of the Butane configuration in subsequent steps.
+
+. Create a machine config to apply the IPsec configuration to your cluster:
+
+.. To add the IPsec configuration, create Butane config files for control plane and worker nodes with the following contents:
++
+[source,terminal,subs="attributes+"]
+----
+$ for role in master worker; do
+  cat >> "99-ipsec-$\{role}-endpoint-config.bu" <<-EOF
+  variant: openshift
+  version: {product-version}.0
+  metadata:
+    name: 99-$\{role}-import-certs-enable-svc-os-ext
+    labels:
+      machineconfiguration.openshift.io/role: $role
+  openshift:
+    extensions:
+      - ipsec
+  systemd:
+    units:
+    - name: ipsec-import.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Import external certs into ipsec NSS
+        Before=ipsec.service
+      
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/local/bin/ipsec-addcert.sh
+        RemainAfterExit=false
+        StandardOutput=journal
+
+        [Install]
+        WantedBy=multi-user.target
+    - name: ipsecenabler.service
+      enabled: true
+      contents: |
+        [Service]
+        Type=oneshot
+        ExecStart=systemctl enable --now ipsec.service
+
+        [Install]
+        WantedBy=multi-user.target
+  storage:
+    files:
+    - path: /etc/pki/certs/ca.pem
+      mode: 0400
+      overwrite: true
+      contents:
+        local: ca.pem
+    - path: /etc/pki/certs/left_server.p12
+      mode: 0400
+      overwrite: true
+      contents:
+        local: left_server.p12
+    - path: /etc/pki/certs/right_server.p12
+      mode: 0400
+      overwrite: true
+      contents:
+        local: right_server.p12
```

Comment:
@jab-rh We don't need right_server.p12 code part here as communicated in latest butance config earlier. Just left_server part is needed for OCP. Thanks

Reply:
@anuragthehatter, great, I've made that change, thanks!

Reply:
@jab-rh For some reason I still see `right_server.p12" code present in bu config?